CMakeLists.txt

```diff
@@ -19,18 +19,22 @@ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 ########################################
 # Define basic flags
 ########################################
-option(BUILD_TESTS     "Build deepworks with tests"           ON)
-option(WITH_EIGEN      "Build deepworks with eigen backend"   ON)
-option(BUILD_SAMPLES   "Build samples"                        ON)
-option(BUILD_BENCHMARKS "Build benchmarks"                    ON)
-
+option(BUILD_TESTS      "Build deepworks with tests"         ON)
+option(WITH_EIGEN       "Build deepworks with eigen backend" ON)
+option(BUILD_SAMPLES    "Build samples"                      ON)
+option(BUILD_BENCHMARKS "Build benchmarks"                   ON)
+option(DOWNLOAD_DATA    "Download and unpack datasets"       ON)
```

Comment:
Please add  `DOWNLOAD_DATA` to README.md

Reply:
Done